habitat-lab/habitat/core/utils.py

```diff
@@ -92,6 +101,10 @@ def try_cv2_import():
 
 
 class Singleton(type):
```

Comment:
is "type" a correct class parameter?  The HTML documentation is pointing the the standard library and showing dunders from the built in library.

Reply:
I think this is fine. This metaclass is extending the base object "type" factory.